src/blockchain/Blockchain.ts

```diff
@@ -183,7 +183,7 @@ export default class Blockchain implements IBlockchainApi {
     address: KeyringPair,
     accumulator: Accumulator
   ): Promise<void> {
-    const update = await this.api.tx[this.chainmod].updateAccumulator(
```

Comment:
Removing this `await` results in multiple test errors since you would be calling signAndSend on a promise.